contrib/libunwind-cmake/unwind-override.c

```diff
@@ -1,6 +1,10 @@
 #include <libunwind.h>
 
+#if defined(__APPLE__)
+int backtrace(void ** buffer, int size);
+#else
 int backtrace(void ** buffer, int size)
 {
     return unw_backtrace(buffer, size);
```

Comment:
Why exactly it doesn't work on Macintosh?

Reply:
`unw_backtrace` is a function from libunwind, but I would love `backtrace` to be used from dynamically linked library. 

Moreover `exception` header from libcxx in our contrib was patched to use `unw_backtrace` and thus libunwind, so I patched it to use `backtrace`.

Reply:
It is prohibitive on Linux with glibc.
glibc uses an idiotic implementation: - it locates a .so at runtime and does dlopen.

It also does not work on Musl.

But I thought that it could be acceptable on Mac.